### PR TITLE
Bump Sphinx to 2.2.0.

### PR DESCRIPTION
**Pull request by JulienPalard**

### CODE REVIEW

**Review by willingc — APPROVED** [review]
Thanks @JulienPalard. Feel free to merge 👍

**Comment by ned-deily** [comment]
Let's not merge this until after the current round of final releases (3.8.0, 3.7.5, 2.7.17) is out the door, please.   It's better to keep all the branches using the same version of Sphinx as much as possible, particularly on the doc server.

**Review by ned-deily — APPROVED** [review]

**Comment by willingc** [comment]
Sounds good to me @ned-deily. Thanks!

**Pull request merged**
